TLM/TLM/Manager/Impl/VehicleRestrictionsManager.cs

```diff
@@ -363,53 +363,49 @@ public class VehicleRestrictionsManager
         }
 
         /// <summary>
-        /// Restores all vehicle restrictions on a given segment to default state.
+        /// Reset vehicle restrictions for a specific lane on the specified segment.
         /// </summary>
-        /// <param name="segmentId"></param>
-        /// <returns></returns>
-        internal bool ClearVehicleRestrictions(ushort segmentId, byte laneIndex, uint laneId) {
+        /// <param name="segmentId">The id of the segment containing the lane.</param>
+        /// <param name="laneIndex">The index of the lane in the segment.</param>
+        /// <param name="laneId">The id of the lane.</param>
+        internal void ClearVehicleRestrictions(ushort segmentId, byte laneIndex, uint laneId) {
             NetInfo segmentInfo = segmentId.ToSegment().Info;
             NetInfo.Lane laneInfo = segmentInfo.m_lanes[laneIndex];
-            return SetAllowedVehicleTypes(
```

Comment:
The return value never used for both `ClearVehicleRestrictions()` methods so set both as `void`.